printThis.js

```diff
@@ -92,7 +92,7 @@
                 $body = $doc.find("body");
 
             // add base tag to ensure elements use the parent domain
-            $head.append('<base href="' + document.location.protocol + '//' + document.location.host + '">');
+            $head.append('<base href="' + document.location.protocol + '//' + document.location.host + document.location.pathname+'">');
```

Comment:
Have you tested this outside of your app?  I don't believe this will work beyond your use case.

If I set the base href with document.location.pathname, the CSS path & js path will be built on that base, resulting in the paths being wrong. Example - navigate to http://www.mysite.com/blog/entry/if-i-was-a-rich-man/ - my pathname is "/blog/entry/if-i-was-a-rich-man/", but my CSS & js paths are not subdirectories of the pathname, but at the root url.